src/components/Navbar.js

```diff
@@ -45,31 +53,40 @@ const Nav = styled.nav`
   gap: 60px;
 `
 
-const ObservationIconWrapper = styled.div`
+const IconWrapper = styled.div`
   width: 100%;
   display: grid;
   grid-template-columns: 1fr 1fr;
   justify-items: space-between;
   padding-left: 30px;
+  align-items: center;
 `
 
 const MedicationsIconStyled = styled(MedicationsIcon)`
   stroke: ${prop =>
-    prop.$isActive ? 'var(--color-quaternary)' : 'var(--color-primary)'};
-  width: 33px;
-  height: 60px;
+    prop.$isActive ? 'var(--color-gradient-2)' : 'var(--color-primary)'};
+  width: 30px;
+  height: 40px;
+`
```

Comment:
You can put the logic in a var in the globalstyle.css so its a bit more cleaner like this
``` --iconStroke: ${prop => prop.$isActive ? 'var(--color-quaternary)' : 'var(--color-primary)'}```

Reply:
I like that!!